internal/configuration.go

```diff
@@ -0,0 +1,17 @@
+package internal
+
+import "github.com/caarlos0/env/v11"
```

Comment:
Add comments to the struct fields to explain their purpose and what values are expected. This will improve the readability and maintainability of the code.